compiler/src/iree/compiler/Modules/IO/Parameters/Transforms/RuntimeUtils.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 The IREE Authors
```

Comment:
maybe ArchiveUtils or something? "runtime" implies dealing with execution, but this is just wrapping a parameter archive utility thing